package-mixins.js

```diff
@@ -1,4 +1,13 @@
-var { NativeAppEventEmitter } = require("react-native");
+var Platform = require("Platform");
+var EventEmitter;
+
+if (Platform.OS === "android") {
+  var { DeviceEventEmitter } = require("react-native");
+  EventEmitter = DeviceEventEmitter;
+} else if (Platform.OS === "ios") {   
+  var { NativeAppEventEmitter } = require("react-native");
+  EventEmitter = NativeAppEventEmitter;
```

Comment:
I thought iOS supports `DeviceEventEmitter` as well. Could we use that for both?